training_scripts/gsgnn_np/README.md

```diff
@@ -1,10 +1,46 @@
-# Node Classification Example
+# Training script examples for node classification/regression
+This folder provides example yaml configurations for node classification and regression training tasks.
+The configurations include:
 
-# Movielens node classification
-Preparing movielens dataset for node classification.
+  * ``ml_nc.yaml`` defines a node classification task on the ``movie`` nodes. The target label field is ``label``. It uses a single-layer RGCN model as its graph encoder.
 
-## Preparation
-Coming soon.
+  * ``ml_nr.yaml`` defines a node regression task on the ``user`` nodes. The target label field is ``age``. It uses a single-layer RGCN model as its graph encoder.
 
-## Training
-Coming soon.
+  * ``ml_nc_homogeneous.yaml`` defines a node classification task a homogeneous graph. The target label field is ``label``. It uses a single-layer GraphSage model as its graph encoder.
+
+  * ``ml_nc_utext.yaml`` defines a node classification task on the ``movie`` nodes. The target label field is ``label``. It uses a single-layer RGCN model as its graph encoder. In addition, the training task will do **LM-GNN co-training**. A BERT model, i.e., ``bert-base-uncased``, is used to compute the text embeddings of ``movie`` nodes and ``user`` nodes on the fly. During training, GraphStorm will randomly select 10 nodes for each mini-batch to participate the gradient computation to tune the BERT models. For more detials, please refer to https://graphstorm.readthedocs.io/en/v0.3.1/advanced/language-models.html#fine-tune-lms-on-graph-data.
+
+  * ``ml_nc_movie_utext.yaml`` defines a node classification task on the ``movie`` nodes. The target label field is ``label``. Similar to ``ml_nc_utext.yaml``, it also defines a LM-GNN co-training task. But only the the text embeddings of ``movie`` nodes are computed using the co-trained BERT model.
+
+  * ``ml_nc_utext_roberta.yaml`` defines a node classification task on the ``movie`` nodes. The target label field is ``label``. Similar to ``ml_nc_utext.yaml``, it also defines a LM-GNN co-training task. But it uses a RoBERTa model instead of a BERT model.
```

Comment:
```suggestion
  * ``ml_nc_utext_roberta.yaml`` defines a node classification task on the ``movie`` nodes. The target label field is ``label``. Similar to ``ml_nc_utext.yaml``, it also defines a **LM-GNN co-training** task. But it uses a RoBERTa model instead of a BERT model.
```